src/apps/content-editor/src/app/views/ItemList/ItemList.js

```diff
@@ -56,708 +63,630 @@ export default connect((state, props) => {
         state.content[item].meta.ZUID.slice(0, 3) !== "new" // Don't include new items
     )
   };
-})(
-  class ItemList extends Component {
-    _isMounted = false;
-
-    // We use this ref to determine the list dimensions
-    table = React.createRef();
-    list = React.createRef();
-
-    state = {
-      saving: false,
-      loading: false,
-      loadingSecondary: false,
-      // Keep internal item state for handling table filters performantly
-      // avoiding recalculations on component updates
-      items: [],
-      fields: [],
-      height: 1000,
-      width: 1000,
-      sortedBy: null,
-      status:
-        (this.props.filters[this.props.modelZUID] &&
-          this.props.filters[this.props.modelZUID].status) ||
-        "all"
+})(function ItemList(props) {
+  const history = useHistory();
+  const _isMounted = useRef(false);
+
+  // table, list dimensions
+  const table = useRef();
+  const list = useRef();
+  const [height, setHeight] = useState(1000);
+  const [width, setWidth] = useState(1000);
+
+  // saving, loading states
+  const [saving, setSaving] = useState(false);
+  const [loadingFirstPage, setLoadingFirstPage] = useState(false);
+  const [loadingSecondaryPages, setLoadingSecondaryPages] = useState(false);
```

Comment:
I am still not clear on these. What is a page? An app screen or an API page? Would something like `initialLoad` and `backgroundLoad` explain what these deal with?